vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/ProjectViewPage.java

```diff
@@ -38,30 +39,35 @@ public class ProjectViewPage extends Div implements BeforeEnterObserver, RouterL
 
   public ProjectViewPage(@Autowired ProjectNavigationBarComponent projectNavigationBarComponent,
       @Autowired ProjectInformationPage projectInformationPage,
-      @Autowired ExperimentInformationPage experimentInformationPage) {
+      @Autowired ExperimentInformationPage experimentInformationPage,
+      @Autowired SampleInformationPage sampleInformationPage) {
     Objects.requireNonNull(projectNavigationBarComponent);
     Objects.requireNonNull(projectInformationPage);
     Objects.requireNonNull(experimentInformationPage);
     add(projectNavigationBarComponent);
-    setPageStyles(projectNavigationBarComponent, projectInformationPage, experimentInformationPage);
+    setPageStyles(projectNavigationBarComponent, projectInformationPage, experimentInformationPage,
+        sampleInformationPage);
     handler = new ProjectViewHandler(projectNavigationBarComponent, projectInformationPage,
-        experimentInformationPage);
+        experimentInformationPage, sampleInformationPage);
     log.debug(String.format(
-        "New instance for project view (#%s) created with a project navigation component (#%s), a project information page (#%s) and experiment information page (#%s)",
+        "New instance for project view (#%s) created with a project navigation component (#%s), a project information page (#%s), an experiment information page (#%s), and a sample information page (#%s)",
         System.identityHashCode(this), System.identityHashCode(projectNavigationBarComponent),
         System.identityHashCode(projectInformationPage),
-        System.identityHashCode(experimentInformationPage)));
+        System.identityHashCode(experimentInformationPage),
```

Comment:
what does this do?

Reply:
This logs purpose is to check if the subPages were created. identityHash code ensures that the default hash code for the given object is returned independent of if the given object's class overrides hashCode()